source/OutfitterPanel.cpp

```diff
@@ -888,10 +888,7 @@ bool OutfitterPanel::IsLicense(const string &name) const
 	static const string &LICENSE = " License";
 	if(name.length() < LICENSE.length())
 		return false;
-	if(name.compare(name.length() - LICENSE.length(), LICENSE.length(), LICENSE))
-		return false;
-
-	return true;
+	return !name.ends_with(LICENSE);
```

Comment:
If something is a license, it should end with the keyword, no need to flip it.
```suggestion
	return name.ends_with(" License");
```